BUILD.bazel

```diff
@@ -13,3 +16,16 @@ cc_capnp_library(
 refresh_compile_commands(
     name = "refresh_compile_commands",
 )
+
+npm_link_all_packages(name = "node_modules")
```

Comment:
This will break our main repo until we add @aspect_rules_js and @npm rules to it.